packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -27,7 +30,20 @@ const styles = {
 };
 
 const GlobalHeader = ({ editUrl, className, search }) => {
-  const [isOverlayOpen, setIsOverlayOpen] = useState(false);
+  const [isOverlayOpen, setIsOverlayOpen] = useState(null);
```

Comment:
You may consider setting the initial state based on whether the query string parameter exists. If someone shared a link or wanted to link to the site with a pre-filled search term, we'd want to open the overlay with the search term pre-filled.

Reply:
the problem is that when I try to check for the query string, the global header has already rendered, so I have to do it in`useEffect` in order to check for it!